### PR TITLE
line 4 mark user

### PR DESCRIPTION
**Pull request by markandeyuluR**

### CODE REVIEW

File: day1 [hunk 1]
```diff
@@ -1 +1,4 @@
 line 1 suman2mark/monday
+
+
+line 4 added by markandeyuluR user on same branch
```
Comment (suman2mark) [inline day1:4]: please add line 6

**Review by suman2mark — COMMENTED** [review]
add some code line6

File: day1 [hunk 2]
```diff
@@ -2,3 +2,6 @@ line 1 suman2mark/monday
 
 
 line 4 added by markandeyuluR user on same branch
+
+added new line 6 .. please merge
```
Comment (suman2mark) [inline day1:6]: add line8

**Comment by markandeyuluR** [comment]
please merge line 6 8

**Comment by markandeyuluR** [comment]
please merge

**Pull request merged**